### ISSUE TITLE
Be able to disable spawns in game lobby

### ISSUE THREAD

**Issue by kyrreso** [OpenRA/OpenRA#18298]
This could be a junior job I think:

You should be able to disable spots in game lobby, so playing with random spots, you don't have any there. 
This is my idea:
![bedre](https://user-images.githubusercontent.com/526956/84964773-45aedb80-b10d-11ea-93b2-7f71f8a749c0.png)

**Comment by ocdi** [comment]
Hi guys, I am new to developing on OpenRA and thought I would have a crack at this task as it seemed achievable for me. I have a working prototype with the td mod. Need to apply the UI changes to the other mods in this repo and then I will create a PR. 

This is how it looks.
![image](https://user-images.githubusercontent.com/2775804/87731426-cd236380-c80d-11ea-99a6-a03091f3b854.png)

I'm not sure how to draw a X, like the above mockup, but I don't think that matters. I picked white as that is less likely to be used, however open to suggestions on this or advise on how to actually render something.

I've got my WIP branch over here if anyone is interested. I will likely create a PR tomorrow for this.

https://github.com/OpenRA/OpenRA/compare/bleed...ocdi:feature/closed-spawn-slots
